presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMergeStatisticalDigestFunction.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.block.BlockEncodingManager;
+import com.facebook.presto.metadata.FunctionManager;
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.sql.analyzer.FeaturesConfig;
+import com.facebook.presto.type.TypeRegistry;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.function.BiFunction;
+
+import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+
+public abstract class TestMergeStatisticalDigestFunction
+        extends AbstractTestAggregationFunction
+{
+    @BeforeClass
+    @Override
+    public void initTestAggregationFunction()
```

Comment:
I'd prefer we avoid this override and keep the parent class'  version final. You don't need this if you don't add enable tdigest in `FeaturesConfig`.